assets/src/scripts/components/Layout/MetadataTab.tsx

```diff
@@ -1,7 +1,10 @@
+import { QueryClient, QueryClientProvider } from "@tanstack/react-query";
+import { ReactQueryDevtools } from "@tanstack/react-query-devtools";
```

Comment:
Ah - so yes in it's current form we need the devtools in `dependencies`. Is there a way to only include it for local development? Or is this the best practice way?

Reply:
> By default, React Query Devtools are only included in bundles when `process.env.NODE_ENV === 'development'`, so you don't need to worry about excluding them during a production build.

It freaks me out every time, [but the docs say this is the way to do it](https://tanstack.com/query/v5/docs/framework/react/devtools).